uefi/src/proto/network/snp.rs

```diff
@@ -186,13 +186,36 @@ impl SimpleNetwork {
         status.to_result_with_val(|| NonNull::new(tx_buf.cast()))
     }
 
-    /// Place a packet in the transmit queue of a network interface.
+    /// Place a packet in the transmit queue of the network interface.
+    ///
+    /// The packet structure depends on the type of network interface, but
+    /// effectively this is always a (wired) ethernet interface. In these cases,
+    /// this function transmits ethernet frames.
```

Comment:
Is there some citation we can point to for this paragraph? In particular, is true that it's always wired? (I wouldn't be terribly surprised if some enterprising motherboard manufacturers supported wifi netboot, for example.)

Reply:
From my understanding, WiFis packet structure makes it incompatible with the interface.

Therefore, we have `EFI_WIRELESS_MAC_CONNECTION_PROTOCOL`. Would you prefer it if I rephrase it as "in most cases, this refers to an Ethernet interface"?

Reply:
Ah, I didn't know that a wifi interface would be incompatible. "in most cases, this refers to an Ethernet interface" sounds good to me